pkg/model/model.go

```diff
@@ -16,13 +16,18 @@ package model
 
 import (
 	trpb "github.com/opensergo/opensergo-control-plane/pkg/proto/transport/v1"
+	"k8s.io/apimachinery/pkg/types"
 )
 
 type NamespacedApp struct {
 	Namespace string
 	App       string
 }
 
+func (st NamespacedApp) String() string {
+	return st.Namespace + string(types.Separator) + st.Namespace
+}
+
```

Comment:
1. func is used.   
2. Namespace is double printed.
3. maybe use join str with json format is better.